central/alert/datastore/datastore_sac_test.go

```diff
@@ -141,71 +117,26 @@ func (s *alertDatastoreSACTestSuite) TestUpsertAlert() {
 	s.testAlertIDs = append(s.testAlertIDs, alert1.Id)
 	s.testAlertIDs = append(s.testAlertIDs, alert2.Id)
 
-	cases := map[string]crudTest{
-		"(full) read-only cannot upsert": {
-			scopeKey:      testutils.UnrestrictedReadCtx,
-			expectError:   true,
-			expectedError: sac.ErrResourceAccessDenied,
-		},
-		"full read-write can upsert": {
-			scopeKey:      testutils.UnrestrictedReadWriteCtx,
-			expectError:   false,
-			expectedError: nil,
-		},
-		"full read-write on wrong cluster cannot upsert": {
-			scopeKey:      testutils.Cluster1ReadWriteCtx,
-			expectError:   true,
-			expectedError: sac.ErrResourceAccessDenied,
-		},
-		"read-write on wrong cluster and wrong namespace name cannot upsert": {
-			scopeKey:      testutils.Cluster1NamespaceAReadWriteCtx,
-			expectError:   true,
-			expectedError: sac.ErrResourceAccessDenied,
-		},
-		"read-write on wrong cluster and matching namespace name cannot upsert": {
-			scopeKey:      testutils.Cluster1NamespaceBReadWriteCtx,
-			expectError:   true,
-			expectedError: sac.ErrResourceAccessDenied,
-		},
-		"read-write on right cluster but wrong namespaces cannot upsert": {
-			scopeKey:      testutils.Cluster2NamespacesACReadWriteCtx,
-			expectError:   true,
-			expectedError: sac.ErrResourceAccessDenied,
-		},
-		"full read-write on right cluster can upsert": {
-			scopeKey:      testutils.Cluster2ReadWriteCtx,
-			expectError:   false,
-			expectedError: nil,
-		},
-		"read-write on the right cluster and namespace can upsert": {
-			scopeKey:      testutils.Cluster2NamespaceBReadWriteCtx,
-			expectError:   false,
-			expectedError: nil,
-		},
-		"read-write on the right cluster and at least the right namespace can upsert": {
-			scopeKey:      testutils.Cluster2NamespacesABReadWriteCtx,
-			expectError:   false,
-			expectedError: nil,
-		},
-	}
+	testedVerb := "upsert"
```

Comment:
Shall we make the `testedVerb | upsert` value as a const of the `testutils` package? Seems like the best solution IMO.